src/Algebra/Graph/Acyclic/AdjacencyMap/Ord.hs

```diff
@@ -1,42 +1,42 @@
------------------------------------------------------------------
+-----------------------------------------------------------------------------
 -- |
--- Module     : Algebra.Graph.Acyclic.Ord
+-- Module     : Algebra.Graph.Acyclic.AdjacencyMap.Ord
+-- Copyright  : (c) Andrey Mokhov 2016-2019
 -- License    : MIT (see the file LICENSE)
+-- Maintainer : andrey.mokhov@gmail.com
 -- Stability  : experimental
 --
+--
 -- __Alga__ is a library for algebraic construction and manipulation of graphs
 -- in Haskell. See <https://github.com/snowleopard/alga-paper this paper> for the
 -- motivation behind the library, the underlying theory, and implementation details.
 --
--- This module defines functions which guarantee Acyclic graph by
--- establishing an invariant on the edges.
--- An edge (x, y) exists if and only if x < y. This way no back
--- edges are formed and hence making a cyclic graph is impossible.
-----------------------------------------------------------------
-module Algebra.Graph.Acyclic.Ord (
-    -- * Graph construction primitives
-    edge, overlay, connect, edges, overlays, connects,
-
-    -- * Graph transformation
-    replaceVertex, mergeVertices, gmap,
+-- This module provides convenient functions for constructing acyclic graphs by
+-- keeping only edges @(x,y)@ where @x < y@ according to the supplied 'Ord' @a@
+-- instance. This module can be imported qualified to avoid confusion with the
+-- modules "Algebra.Graph.AdjacencyMap" and "Algebra.Graph.Acyclic.AdjacencyMap":
+--
+-- @
+-- import qualified Algebra.Graph.Acyclic.AdjacencyMap.Ord as Acyclic.Ord
+-- @
+-----------------------------------------------------------------------------
+module Algebra.Graph.Acyclic.AdjacencyMap.Ord (
+    -- * Basic graph construction primitives
+    edge, overlay, connect, edges, overlays, connects
     ) where
 
 import Algebra.Graph.Acyclic.AdjacencyMap
+
 import qualified Algebra.Graph.AdjacencyMap as AM
-import Data.Function (on)
 
 -- | Construct the graph comprising /a single edge/.
 -- Complexity: /O(1)/ time, memory.
 --
 -- @
 -- edge x y               == 'connect' ('vertex' x) ('vertex' y)
+-- 'hasEdge' 1 1 (edge 1 1) == False
 -- 'hasEdge' 1 2 (edge 1 2) == True
 -- 'hasEdge' 2 1 (edge 2 1) == False
--- 'edgeCount'   (edge 1 2) == 1
```

Comment:
The test cases for this exist even though the examples were removed.

Reply:
Thanks, indeed, the examples and tests have got out of sync. Will fix this!